.github/workflows/admin.yml

```diff
@@ -0,0 +1,107 @@
+name: admin
+
+on: push
+
+defaults:
+  run:
+    working-directory: admin
+
+env:
+  MONGO_URI: temp
+  GOOGLE_ID: temp
+  GOOGLE_SECRET: temp
+  NEXTAUTH_URL: temp
+  ADMIN_EMAIL: temp
```

Comment:
Are these just placeholders since the env vars aren't needed for CI?

Reply:
Yeah, the build just requires that they are non-null afaik (we might actually be ok without them but this was to be safe(
If we were to add tests or anything which had runtime usage we'd need to use the actual values